tests/test_utils/test_transactiontestcase.py

```diff
@@ -79,6 +81,33 @@ def test_queries_cleared(self):
         for alias in connections:
             self.assertEqual(len(connections[alias].queries_log), 0, 'Failed for alias %s' % alias)
 
+    def test_multi_db_deprecation(self):
```

Comment:
I'm in favor of putting the deprecation tests in a separate file to ease removal.

Reply:
Will do.